README.md

```diff
@@ -117,14 +117,14 @@ To run the project, execute the following command:
 ## Contributing
 
 Contributions, issues, and feature requests are welcome!
-Feel free to check the [issues page](https://github.com/SekoViper/OOP-school-library.git/issues).
+Feel free to check the [issues page](https://github.com/SekoViper/OOP-school-library/issues).
 
 ## Show your support
 
 Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./LICENSE) licensed.
+This project is [MIT](https://choosealicense.com/licenses/mit) licensed.
```

Comment:
- [ ] The link should send the user to your MIT.md file and don't forget in your MIT.md file to include all the information about your site such as [APP TYPE]